CMSgov/design-system
# [WNMGDS-889] Change overviews to index

## Summary
- This continues to work to make the top level Overview pages of the side navigation at `/pattterns/` and `/utilities/` instead of `/patterns/overview` and `/utilities/overview`

## How to test
- Visit http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-889/top-level-pages and make sure Overview links work correctly for patterns and utilities
